### DISCUSSION TITLE
Stripe dynamic badge

### DISCUSSION THREAD

**Discussion by amonam** [Glench/ExtPay#122] [General]
How can we display dynamic badges for carbon removal contributions in the checkout popup?

Here is an example form Stripe:
"UV Weather will contribute 1% of your purchase to remove CO₂ from the atmosphere."
![Screenshot 2023-04-12 at 2 17 08 PM](https://user-images.githubusercontent.com/19654690/231548914-1a199970-48f4-4af6-86f8-81f15093f6e8.png)

**Comment by Glench** [comment]
There is no way to do that at the moment, but hang tight :)
